vagrant-provision.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# prepare Debian system
+apt-get update && apt-get dist-upgrade -y
+apt-get install python-pip python-dev build-essential default-jre libxslt-dev libxml2-dev libffi-dev libssl-dev git curl -y
+
+# get nodejs from external sources (Debian is outdated)
+curl -sL https://deb.nodesource.com/setup_6.x | sudo bash -
```

Comment:
Fully agree, however, this is just a virtual machine that you throw away anyways. Therefore, I honestly don't care about executing something from the internet there ;-)